grading/github.py

```diff
@@ -27,8 +27,14 @@ def check_student_repo_exists(org, course, student, token=None):
         repository = "{}-{}".format(course, student)
         g.repository(org, repository)
 
-    except Exception as e:
-        raise e
+    except gh3.exceptions.NotFoundError as e:
```

Comment:
Here is my update. note i commented out the original exception. it's too generic to actually return a custom message. i'm not sure however if we need a "continue" to ensure it keeps running through the students. please advise